tests/unit/states/test_file.py

```diff
@@ -870,7 +870,7 @@ def test_directory(self):
                             'comment': comt,
                             'result': None,
                             'pchanges': p_chg,
-                            'changes': {}
+                            'changes': {name: {'directory': 'new'}}
```

Comment:
I think we just want to skip this change.

https://jenkinsci.saltstack.com/job/pr-kitchen-ubuntu1604-py2/job/PR-49282/1/testReport/junit/unit.states.test_file/TestFileState/test_directory/